R/functions__whenthen.R

```diff
@@ -1,57 +1,56 @@
-#' when-then-otherwise Expr
+#' Make a then-when-otherwise expression
+#'
 #' @name Expr_when_then_otherwise
-#' @description Start a “when, then, otherwise” expression.
-#' @keywords Expr
-#' @param ... Into Expr into a boolean mask to branch by.
-#' @param statement Into Expr value to insert in when() or otherwise().
-#' Strings interpreted as column.
-#' @return Expr
-#' @aliases when then otherwise When Then ChainedWhen ChainedThen
-#' @details
 #'
-#' when-then-otherwise is similar to R `ifelse()`. `pl$when(condition)` takes a condition as input
-#' this will an polars `<Expr>` which renderes to a Boolean column. Then it is chained with a
-#' `$then(statement)` when arg statement is an `<Expr>` which produces a column with values if
-#' idealy all Boolean are true. Then finally an `$otherwise(statement)` with values if false.
-#' `$otherwise()` returns an `Expr` which will mix the `$then()` statement with the `$otherwise()`
-#' as given by the when-condition.
+#' @description
+#' Start a “when, then, otherwise” expression.
 #'
-#' State-machine details below. The state machine consists of 4 classes `<When>`, `<Then>`,
-#' `<ChainedWhen>` & `<ChainedThen>` and a starter function `pl$when()` and the final expression
-#' class  a polars `<Expr>`.
+#' @param ... Expr or something coercible to an Expr into a boolean mask to
+#'   branch by.
+#' @param statement Expr or something coercible to an Expr value to insert in
+#'   when() or otherwise(). Strings interpreted as column.
```

Comment:
This is just used by `roxygen2` to generate the params, it doesn't appear in the man page

Reply:
This is just used by `roxygen2` to generate the params, it doesn't appear in the man page

Reply:
> This is just used by `roxygen2` to generate the params, it doesn't appear in the man page

What do you mean, Link is also working in the params section, right?

Reply:
Oh, sorry. These are on the same page so the link doesn't make sense.

Reply:
I don't why but in the review page your comment appear on the line below (on `@inheritParams Expr_alias`), hence this comment.

But in any case `when()` and `otherwise()` point to the same page so there's no need to link them